.gitignore

```diff
@@ -36,3 +36,4 @@
 
 /app/assets/builds/*
 !/app/assets/builds/.keep
+/byebug_history
```

Comment:
```suggestion
.byebug_history
```

Reply:
Maybe you should delete the file and add its name to `.gitignore` in the same commit.